internal/cmd/db_transfer.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/chiselstrike/iku-turso-cli/internal"
+	"github.com/chiselstrike/iku-turso-cli/internal/prompt"
+	"github.com/chiselstrike/iku-turso-cli/internal/settings"
+	"github.com/chiselstrike/iku-turso-cli/internal/turso"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	dbCmd.AddCommand(dbTransferCmd)
```

Comment:
This will be rarely used. I'd rather not give it the same relevant as `db create`, `db replicate` and so on.

Have you considered adding it under `turso org`?
Something like `turso org db-transfer`.

@penberg thoughts?

Reply:
Looks good to me! Semantically I thought it would be better at first glance to be a subcmd of db, but the usability perspective definitely has more weight, I think.